app/controllers/options_controller.rb

```diff
@@ -1,26 +1,29 @@
 # frozen_string_literal: true
 class OptionsController < ApplicationController
+  include Accessible
 
   def show
     @option = Option.find(params[:id])
   end
 
   def edit
     @option = Option.find(params[:id])
+    execute_if_accessible(subject: @option)
```

Comment:
I don't like this too much — looks like a merge error or something. One would expect the block to follow (the method is called `execute` after all!), but there is none.
Maybe let's alias `execute_if_accessible` as `check_accessibility`?